tests/foreman/cli/test_role.py

```diff
@@ -458,6 +458,8 @@ def test_negative_update_role_taxonomies(self):
 
         :expectedresults: The taxonomies of overridden filter should not be
             updated with role taxonomies
+
+        :CaseAutomation: notautomated
```

Comment:
Since all cases in `CannedRoleTestCases` class are stubbed, you can as well set `:CaseAutomation: notautomated` on class level.

Of course setting it for each test individually is working, too.

Reply:
You are right, of course - I have completey forgot that beside module-level and test-level it is possible to set class-level metadata as well. Fixed.